lib/paranoia.rb

```diff
@@ -284,8 +284,8 @@ def paranoia_sentinel_value
 module ActiveRecord
   module Validations
     module UniquenessParanoiaValidator
-      def build_relation(klass, table, attribute, value)
-        relation = super(klass, table, attribute, value)
+      def build_relation(klass, *args)
+        relation = super
```

Comment:
As far as I know `super` passes exact same arguments, while `super()` would pass none. (And of course `super(klass)` would pass only class, etc.